### PR TITLE
Update for python 3

### PR DESCRIPTION
**Pull request by stephenworsley**
Fixes kapture so that it works in Python 3.

### CODE REVIEW

**Comment by bjlittle** [comment]
Closes https://github.com/SciTools-incubator/kapture/issues/1

File: kapture/wrap_user.py [hunk 1]
```diff
@@ -193,8 +200,8 @@ def usage():
             usage()
         command = sys.argv[2]
         sys.argv = ['-c'] + sys.argv[3:]
-        exec command in {'__doc__': None, '__name__': '__main__',
-                         '__package__': None}
+        exec(command in {'__doc__': None, '__name__': '__main__',
+                         '__package__': None})
```
[the remark this answers is not in the document]
Comment (stephenworsley) [reply kapture/wrap_user.py]: I've addressed this here #9.

**Comment by rhattersley** [comment]
Any reason not to merge?

**Comment by bjlittle** [comment]
@rhattersley LGTM

Shame we've not got any test coverage, see https://github.com/SciTools-incubator/kapture/issues/3 😜

But I might just doppelganger test Python2 and Python3 and eye ball the results...

**Comment by rhattersley** [comment]
> doppelganger test

I think you've just invented a new term! 👍 😁

**Comment by bjlittle** [comment]
LGTM :smile:

**Pull request merged**